xds/src/main/java/com/linecorp/armeria/xds/ParsedResourcesHolder.java

```diff
@@ -20,20 +20,20 @@
 import java.util.Map;
 import java.util.Set;
 
-final class ParsedResourcesHolder {
-    private final Map<String, AbstractResourceHolder> parsedResources;
+final class ParsedResourcesHolder<T> {
+    private final Map<String, T> parsedResources;
```

Comment:
What do you think of adding more constrains to the type parameter?
```suggestion
final class ParsedResourcesHolder<T extends Message> {
    private final Map<String, ResourceHolder<T>> parsedResources;
```